resources/dicts/events/death/general.json

```diff
@@ -1116,7 +1116,7 @@
         ],
         "tags": [],
         "weight": 20,
-        "event_text": "m_c has always wondered where {PRONOUN/m_c/subject} came from. {PRONOUN/m_c/subject/CAP} {VERB/m_c/sneak/sneaks} out of camp, intending to find loners who might have some clues about {PRONOUN/m_c/poss} origins. Frantic with worry and confusion after finding m_c's empty nest in the morning, c_n launches a search. They find {PRONOUN/m_c/poss} cooling body after a long search. They wonder why {PRONOUN/m_c/subject} decided to leave the safety of c_n, and what killed {PRONOUN/m_c/object}.",
+        "event_text": "m_c sneaked out of camp, intending to find more about {PRONOUN/m_c/poss} origins. c_n found {PRONOUN/m_c/poss} cooling body in the morning, after a long search.",
```

Comment:
just for readability i might recommend a couple edits;

> intending to find **out** more about {PRONOUN/m_c/poss} origins

or

> intending to **discover** more about {PRONOUN/m_c/poss} origins

‎ 
this one is optional but might be good;
> **After a long search in the morning,** c_n found {PRONOUN/m_c/poss} cooling body.